internal/featuresupport/features.go

```diff
@@ -95,10 +95,10 @@ func (feature *SnoFeature) getSupportLevel(filters SupportLevelFilters) models.S
 	if !isFeatureCompatibleWithArchitecture(feature, filters.OpenshiftVersion, swag.StringValue(filters.CPUArchitecture)) {
 		return models.SupportLevelUnavailable
 	}
-	if isNotSupported, err := common.BaseVersionLessThan("4.8", filters.OpenshiftVersion); isNotSupported || err != nil {
+	if isNotSupported, err := common.BaseVersionLessThan("4.9", filters.OpenshiftVersion); isNotSupported || err != nil {
```

Comment:
this doesn't look right. ocp supports/doesn't-support SNO regardless of changes in assisted-service
I think it means this condition is redundant now, because we've got to a state where SNO is supported for all ocp versions that we serve

Reply:
So you think it is safe to remove it? no possible way someone can still install 4.8?